irods_python_ci_utilities/irods_python_ci_utilities.py

```diff
@@ -103,9 +103,9 @@ def install_os_packages(packages):
         raise_not_implemented_for_distribution()
 
 def install_os_packages_from_files_apt(files):
-    # Files are installed individually in the order supplied, so inter-file dependencies must be handled by the caller
-    for f in files:
-        subprocess_get_output(['sudo', 'apt', 'install', '-fy', f], check_rc=True)
+    args = ['sudo', 'apt', 'install', '-fy'] + list(files)
```

Comment:
Should this go inside the `if` block?

Should we leave a comment explaining *why* we are installing all of the package files together?

Reply:
Yes, I'll put in a comment.  Also, mysterious that this wasn't changed sooner: why do the corresponding `*_{dnf,yum}` functions implement the same logic but not `*_apt`, til now?  Was it due to some shortcoming in apt itself in some old distros?